actors/runtime/src/main/java/cloud/orbit/actors/runtime/Execution.java

```diff
@@ -82,8 +82,20 @@ protected void onInvocation(final HandlerContext ctx, final Invocation invocatio
         final LocalObjects.LocalObjectEntry<Object> entry = objects.findLocalObjectByReference(toReference);
         if (entry != null)
         {
-            final Task<Object> result = InternalUtils.safeInvoke(() -> entry.run(
-                    target -> performInvocation(ctx, invocation, entry, target)));
+            Task<Object> result;
+
+            if (invocation.isDeactivate())
+            {
```

Comment:
Not sure if the code from this branch works because of the concurrency. Is it possible that the "performInvoke" is actually executed after the runtime.deactivateActor() and to re-activate the actor?